content/projects/gsoc/2024/project-ideas/buiilding-android-app.adoc

```diff
@@ -0,0 +1,64 @@
+---
+layout: gsocprojectidea
+title: "Building Android Apps with Jenkins"
+goal: "Describe best practices and provide architectural templates for building Android applications with Jenkins"
+category: Tools
+year: 2024
+status: draft
+sig: platform
+skills:
+- Java
+- YAML
+- Android development
+- Command line tools
+- Package management tool theory
+mentors:
+- "gounthar"
+links:
+    emailThread: https://community.jenkins.io/t/gsoc-2023-project-idea-building-android-apps-with-jenkins/4798
+---
+//   gitter: "jenkinsci_plugin-installation-manager-cli-tool:gitter.im"
+//   draft: https://docs.google.com/document/d/1s-dLUfU1OK-88bCj-GKaNuFfJQlQNLTWtacKkVMVmHc
+
+=== Background
+
+Applying CI principles when building IOS applications brings its own challenges, different from those of more classical applications.
+How to proceed, what are the successful patterns, what are the pitfalls is poorly documented. 
+It very often requires potential user and Jenkins Administrator to "reinvent the wheel".
```

Comment:
```suggestion
It very often requires potential users and Jenkins Administrators to "reinvent the wheel".
```
Updating pluralization so that this is more clear.